omnij-cli/src/main/groovy/foundation/omni/cli/ConsensusCLI.groovy

```diff
@@ -116,8 +116,8 @@ class ConsensusCLI extends CliCommand {
 
     void output(ConsensusSnapshot snap, PrintWriter writer, boolean tsv) {
         snap.entries.each { Address address, BalanceEntry entry ->
-            String balance = entry.balance.toPlainString()
-            String reserved = entry.reserved.toPlainString()
+            String balance = entry.balance.toString()
```

Comment:
I haven't really tested the `toString()` behavior of  `OmniValue`, but if I recall correctly, then `toPlainString()` was used here and there to avoid scientific notations.